frontend-nannyway/src/components/header/NavMenuSecondaryColor.js

```diff
@@ -0,0 +1,55 @@
+import {Button, Menu, MenuItem} from "@material-ui/core";
+import React from "react";
+import {Link} from "react-router-dom";
+import {makeStyles} from "@material-ui/core/styles";
+import myTheme from "../../styling/muiTheme";
+
+const useStyles = makeStyles((theme) => ({
+    darkColor: {
+        color: myTheme.palette.secondary.dark,
+    },
+    buttonMenu: {
+        display: "flex",
+        justifyContent: "center",
+    }
+}));
+
+export default function NavMenuSecondaryColor() {
```

Comment:
It's the same issue here as with `<AppBarSecondaryColor>`. You should pass a prop and change the appearance that way instead of making two very similar components. 

```suggestion
export default function NavMenuSecondaryColor({colorScheme}) {
```